protocol/src/main/java/org/geysermc/mcprotocollib/network/tcp/TcpPacketCodec.java

```diff
@@ -9,21 +9,18 @@
 import io.netty.buffer.ByteBuf;
 import io.netty.channel.ChannelHandlerContext;
 import io.netty.handler.codec.ByteToMessageCodec;
+import lombok.RequiredArgsConstructor;
 
 import java.util.List;
 
+@RequiredArgsConstructor
 public class TcpPacketCodec extends ByteToMessageCodec<Packet> {
     private final Session session;
     private final boolean client;
```

Comment:
i'd avoid adding usage of lombok for simple cases like this where the code is already written